src/index.html

```diff
@@ -10,5 +10,39 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/">
+        <img class="logo" src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops$comuters" class="nav__link"
+            data-qa="hover">Laptops &amp; Computers</a>
```

Comment:
```suggestion
            <a 
              href="#laptops$comuters"
              class="nav__link"
              data-qa="hover"
            >
              Laptops &amp; Computers
            </a>
```
Better format it as on example above :arrow_up: